quix-frontend/client/src/lib/runner/services/autocomplete/highlight-and-score.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  createMatchMask,
+  findAllIndexOf,
+} from './autocomplete-utils';
+import { findRelevantPartOfPrefix } from "../../../sql-autocomplete/adapter/sql-autocomplete-adapter-utills";
+import { ICompleterItem as AceCompletion } from '../../../code-editor/services/code-editor-completer';
+import { QueryContext } from '../../../sql-autocomplete/sql-context-evaluator';
+
+const PERFECT_SCORE = 10000;
+
+export function enrichCompletionItemAfterDotObject(all: AceCompletion[]): AceCompletion[] {
+  const indexes = [0];
+  return all.map(completionItem => ({
+    ...completionItem,
+    matchMask: createMatchMask(indexes, 0),
+    score: PERFECT_SCORE,
+  }));
+}
+
+export function enrichCompletionItemInObjectSearch(all: AceCompletion[], queryContext: QueryContext, prefix: string): AceCompletion[] {
+  return all.reduce((resultArr, completionItem) => {
+    const relevantPartOfPrefix = findRelevantPartOfPrefix(queryContext.tables, prefix.split('.')).slice(0, -1);
```

Comment:
which part is considered to be relevant?

Reply:
its the path of the collum and onwards...
do you think `columnPath` is more appropriate ?